docker_reduction/mantid/tests/input/reduce_vars.py

```diff
@@ -0,0 +1,30 @@
+standard_vars = {
```

Comment:
This file doesn't need to committed either for the time being. We are not testing it and it will not be required in most cases